internal/cmd/cmd_decr.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"math"
+
+	"github.com/dicedb/dice/internal/object"
+	dstore "github.com/dicedb/dice/internal/store"
+	"github.com/dicedb/dice/wire"
+)
+
+var cDECR = &DiceDBCommand{
+	Name:      "DECR",
+	HelpShort: "DECR decrements the value of the specified key in args by 1",
+	Eval:      evalDECR,
+}
+
+func init() {
+	commandRegistry.AddCommand(cDECR)
+}
+
+//nolint:gocyclo
+func evalDECR(c *Cmd, s *dstore.Store) (*CmdRes, error) {
+	if len(c.C.Args) <= 1 {
+		return cmdResNil, errWrongArgumentCount("DECR")
+	}
+	incr := int64(-1)
+
+	key := c.C.Args[0]
+	obj := s.Get(key)
+	if obj == nil {
+		obj = s.NewObj(incr, -1, object.ObjTypeInt)
```

Comment:
Create a contant in the `cmd` module called `INFINITE_EXPIRATION` and set the value to be `-1`. Use it here. This way, we will be able to reuse the same variable throughout the codebase.